_news-items/2024-08-16-collins.html

```diff
@@ -0,0 +1,35 @@
+---
+# Copyright 2024 seL4 Project a Series of LF Projects, LLC.
+# SPDX-License-Identifier: CC-BY-SA-4.0
+title: Welcome Collins, Bronze sponsor of the seL4 Summit 2024
+anchor: sponsor24-collins
+---
+
+<p>
+  <a href="https://www.collinsaerospace.com">
+    <img src="../Foundation/Membership/LOGOS/Collins_Aerospace_Logo.svg" style="width: 15%; float:right" alt="Collins Aerospace logo">
+  </a>
+  The seL4 Foundation thanks <a href="https://www.collinsaerospace.com">Collins Aerospace</a> for
+  becoming a Bronze sponsor of the <a href="../Foundation/Summit/2024">seL4
+  Summit 2024</a>.
+</p>
+<p>
+  <a href="https://www.collinsaerospace.com">Collins Aerospace</a>, part of seL4 Foundation
+    member <a href="https://www.rtx.com">Raytheon Technologies</a>, has been a long-time core
+    participant in the seL4 ecosystem.
+  <a href="../Foundation/Summit/2024">
+        <img src="../images/summit/sel4_summit_2024.svg" style="width: 15%; float:right; padding-bottom:5px"
+      alt="seL4 summit">
+  </a>
+  It was a prime contractor in the DARPA HACMS
+  program, which demonstrated the seL4-based incremental cyber retrofit of
+  autonomous military vehicles. This was a major milestone in the growth of seL4,
+  demonstrating that it protects against cyber attacks on real systems in
+  operation. The same team also led the follow-on DARPA CASE program, aiming at
+  designed-in cyber-resiliency, including the seL4-based framework for verified
+  initialisation and configuration of systems architectures.`
```

Comment:
Add: "Collins Aerospace is now leading the INSPECTA team as part of DARPA's PROVERS program, which aimsat developing formal methods tools to guide software engineers through designing proof-friendly software systems and reduce the proof repair workload."